e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -513,6 +531,12 @@
               },
               "TransportType": {
                 "value": "Mqtt"
+              },
+              "TwinUpdateSize": {
+                "value": "<TwinUpdateSize>"
+              },
+              "TwinUpdateFrequency": {
+                "value": "<TwinUpdateFrequency>"
```

Comment:
I added these params to connectivity as we might want to configure them in the future.